gapic/utils/code.py

```diff
@@ -21,3 +23,29 @@ def empty(content: str) -> bool:
     """
     return not any([i.lstrip() and not i.lstrip().startswith('#')
                     for i in content.split('\n')])
+
+
+T = TypeVar('T')
+
+
+def partition(iterator: Iterable[T],
```

Comment:
Recommend flipping the function signature to `predicate, iterable` to match [itertools.filterfalse](https://docs.python.org/3/library/itertools.html#itertools.filterfalse) and others.

Reply:
Will do; my initial rationale was to give the predicate a default value, but consistency with stdlib and builtins is better.